page/news.php

```diff
@@ -7,46 +7,45 @@
-                        <div class="tanggal">2017</div>
-                    </div>
-                </div>
-                <div class="col-md-4">
-                    <h5 class="font-weight-bolder mb-4">
-                        <a href="">Ketua DPD RI Kunjungi Pindad Tinjau Ventilator Dan produk Lainnya Untuk Memerangi Covid-19</a>
-                    </h5>
-                    <div class="deskripsi">
-                        PT Pindad (Persero) memperkenalkan ventilator dan produk-produk lainnya untuk memerangi Covid-19 pada kunjungan kerja Ketua Dewan Perwakilan Daerah RI (DPD), AA La Nyalla Mahmud Mattalitti (20/4). Direktur Utama, Abraham Mose beserta jajaran Direksi menerima kunjungan Ketua DPD dalam rangka menduk...
-                    </div>
-                </div>
-                <div class="col-md-6">
-                    <a href=""><img src="img/berita/1._Penandatanganan_Kontrak_Tank_Boat_2_.jpg" class="img-fluid" alt=""></a>
-                </div>
-            </div>
-            <div class="row article-berita mb-5 pb-5">
-                <div class="col-md-2 text-center">
-                    <div class="tanggal-group float-right">
-                        <div class="hari">21</div>
-                        <div class="tanggal">apr</div>
-                        <div class="tanggal">2017</div>
-                    </div>
-                </div>
-                <div class="col-md-4">
-                    <h5 class="font-weight-bolder mb-4">
-                        <a href="">Ketua DPD RI Kunjungi Pindad Tinjau Ventilator Dan produk Lainnya Untuk Memerangi Covid-19</a>
-                    </h5>
-                    <div class="deskripsi">
-                        PT Pindad (Persero) memperkenalkan ventilator dan produk-produk lainnya untuk memerangi Covid-19 pada kunjungan kerja Ketua Dewan Perwakilan Daerah RI (DPD), AA La Nyalla Mahmud Mattalitti (20/4). Direktur Utama, Abraham Mose beserta jajaran Direksi menerima kunjungan Ketua DPD dalam rangka menduk...
+
+            <?php
+                $select = mysqli_query($connection, "select * from berita ORDER BY tanggal DESC");
+                while($data = mysqli_fetch_array($select)){
+                    if($data[4]==null || $data[4]==""){
+                        $thumbnail = "#";
+                    }else{
+                        $thumbnail = "news/".explode("-", $data[4])[0]."/gambar/".explode("-", $data[4])[1];
+                    }
+                    ?>
+
+                    <div class="row article-berita mb-5 pb-5">
+                        <div class="col-md-2 text-center">
+                            <div class="tanggal-group float-right">
+                                <div class="hari"><?php echo date("d", strtotime($data[2])) ?></div>
+                                <div class="tanggal"><?php echo date("M", strtotime($data[2])) ?></div>
+                                <div class="tanggal"><?php echo date("Y", strtotime($data[2])) ?></div>
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)